### PR TITLE
december monthly email

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: server/controllers/groups.js [hunk 1]
```diff
@@ -127,11 +127,11 @@ export const getTransactions = (req, res, next) => {
     GroupId: req.group.id
   };
 
-  if (req.query.donation) {
+  if (req.query.donation || req.query.type === 'donations') {
```
Comment (xdamman) [inline server/controllers/groups.js]: This is needed for https://github.com/OpenCollective/opencollective-website/pull/617

File: templates/emails/group.monthlyreport.hbs [hunk 2]
```diff
@@ -56,10 +56,10 @@ th {
 </div>
 
 <div>
-  <p><i>Previous email didn't include all the transactions happening on the very last day of the last month (11/30/2016). Here is an updated version of the report.</i></p>
 <p>Hi {{recipient.firstName}}!<br />
-We have lots of good news this month, we could barely wait to share them with you! But first, your monthly stats (visit your collective page to get the latest up to date data: <a href="{{group.publicUrl}}?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">{{group.publicUrl}})</a>.<br />
-And please don't hesitate to send us feedback - just reply to this email! We love to hear from you! ❤️❤️</p>
+We were so happy 2016 had finally ended that we missed sending your December report on time! 🙊 <br/>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: Sounds awkward. "We got so excited about 2016 ending that we missed sending December report on time! 🙊 "

File: templates/emails/group.monthlyreport.hbs [hunk 3]
```diff
@@ -141,38 +141,35 @@ And please don't hesitate to send us feedback - just reply to this email! We lov
...
-<p> </p>
-<p><a href="http://hackerparadise.org?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">Hacker Paradise</a> is offering <b>$100 off</b> their <a href="http://www.hackerparadise.org/blog/2016/11/01/south-america-housing-coworking/?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">upcoming trip to South America</a> ✈️ (simply mention Open Source Collective when registering). You can also apply to get one of their 2 <a href="http://www.hackerparadise.org/blog/2016/11/17/open-source-fellowship/?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">Open Source Fellowships</a> to spend a month working on your open source project in South America, expenses paid!</p>
+<p>You probably noticed that we re-designed the collective’s page. We decided to put the expenses and donations at the top to emphasis the financial transparency aspect of Open Collective and embrace the fact that in order to do great things together, we need money to fund them. We wrote about it <a href="https://medium.com/open-collective/meet-the-open-source-design-collective-96e81261f76#.neu5eimgr?{{utm}}">here</a>! </p>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: s/re-designed/redesigned. 
s/the collective's page/your collective page

File: templates/emails/group.monthlyreport.hbs [hunk 4]
```diff
@@ -141,38 +141,35 @@ And please don't hesitate to send us feedback - just reply to this email! We lov
...
+<ul>
+  <li><a href="https://opencollective.com/telescope?{{utm}}">Telescope</a> is on a mission to create the easiest way to build modern web apps.
+  <li><a href="https://opencollective.com/magit?{{utm}}">Magit</a> is on a mission to make version control at the speed of thought possible with Git.</li>
+  <li><a href="https://opencollective.com/redux-saga?{{utm}}">Redux-saga</a> is on a mission to bring ease for side effects management in Redux applications.</li>
+  <li><a href="https://opencollective.com/veganizerbxl?{{utm}}">VeganizerBXL</a> is on a mission to make Brussels more vegan-friendly</li>
+  <li>Special welcome to <a href="https://opencollective.com/wwcodekyiv?{{utm}}">WWCodeKyiv</a> and <a href="https://opencollective.com/openlabs?{{utm}}">Open Labs Hackerspace</a> our first Ukrainian and Albanian collectives! </li>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: nice!

File: templates/emails/group.monthlyreport.hbs [hunk 5]
```diff
@@ -141,38 +141,35 @@ And please don't hesitate to send us feedback - just reply to this email! We lov
...
+</ul>
 
-<p>Welcome <a href="https://opencollective.com/opensourcedesign?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">Open Source Design</a>! We are thrilled to have you on board our first design collective. Do you care about design in open source? 👉You can now fund it!</p>
+<h2>Read & Listen</h2>
 
-<p>🎉Congrats <a href="https://twitter.com/mxstbr">Max Stoiber</a> from <a href="https://opencollective.com/react-boilerplate?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">React Boilerplate</a> for being <a href="https://twitter.com/mxstbr/status/795555176203620353?utm_source=opencollective&utm_campaign=monthlyreport&utm_medium=email">featured on Forbes Austria</a> about your Open Source work!  </p>
+<p>Check out our <a href="https://medium.com/open-collective/2016-on-open-collective-7677e352fd22#.8rl8hf1mh?{{utm}}">end of year wrap up blog post</a> (and what’s on the works for 2017)</p>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: what's *in* the works...

File: templates/emails/group.monthlyreport.hbs [hunk 6]
```diff
@@ -141,38 +141,35 @@ And please don't hesitate to send us feedback - just reply to this email! We lov
...
 
-<h2>Changelog</h2>
-<p>You can now add a widget to your company or personal website to show the collectives that you are supporting. Simply add the following line of code:</p> <p><code>&lt;script src="https://opencollective.com/:username/widget.js"&gt;&lt;/script&gt;</code></p>
-<p>This is how it looks: <img src="https://cl.ly/1J0Q300v2T12/Screen%20Shot%202016-12-01%20at%204.23.41%20PM.png" width="100%" /></p>
-</ul>
+<p>Aseem kicked-off <a href="https://medium.com/open-collective/meet-the-open-source-design-collective-96e81261f76#.agc4sypg6?{{utm}}">a new series of blog posts to showcase collectives with Open Source Design</a> stay tuned for more to come.</p>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: s/kicked-off/kicked off

Period after the hyperlink and capitalize "Stay..."

File: templates/emails/group.monthlyreport.text.hbs [hunk 7]
```diff
@@ -1,7 +1,7 @@
 Hi {{recipient.firstName}}!
 
-We have lots of good news this month, we could barely wait to share them with you! But first, your monthly stats (visit your collective page to get the latest up to date data: {{group.publicUrl}}).
-And please don't hesitate to send us feedback - just reply to this email! We love to hear from you! ❤️❤️
+We were so happy 2016 had finally ended that we missed sending your December report on time! 🙊
```
Comment (asood123) [inline templates/emails/group.monthlyreport.text.hbs]: same text as above

File: templates/emails/group.monthlyreport.text.hbs [hunk 8]
```diff
@@ -35,46 +35,42 @@ Go to {{@root.group.publicUrl}}/expenses to submit a new expense.
...
- 
-Hacker Paradise (http://hackerparadise.org) is offering $100 off their upcoming trip to South America (http://www.hackerparadise.org/blog/2016/11/01/south-america-housing-coworking/) ✈️ (simply mention Open Source Collective when registering). You can also apply to get one of their 2 Open Source Fellowships to spend a month working on your open source project in South America, expenses paid! More info: http://www.hackerparadise.org/blog/2016/11/17/open-source-fellowship/
+# Changelog
 
+You probably noticed that we re-designed the collective’s page. We decided to put the expenses and donations at the top to emphasis the financial transparency aspect of Open Collective and embrace the fact that in order to do great things together, we need money to fund them. We wrote about it here: https://medium.com/open-collective/meet-the-open-source-design-collective-96e81261f76#.neu5eimgr
```
Comment (asood123) [inline templates/emails/group.monthlyreport.text.hbs]: same comment as above

File: templates/emails/group.monthlyreport.text.hbs [hunk 9]
```diff
@@ -35,46 +35,42 @@ Go to {{@root.group.publicUrl}}/expenses to submit a new expense.
...
 
+# Read & Listen
 
-# Changelog
+Check out our end of year wrap up blog post (and what’s on the works for 2017): https://medium.com/open-collective/2016-on-open-collective-7677e352fd22#.8rl8hf1mh
 
-You can now add a widget to your company or personal website to show the collectives that you are supporting. Simply add the following line of code: 
-<script src="https://opencollective.com/:username/widget.js"></script>
-This is how it looks: https://cl.ly/1J0Q300v2T12/Screen%20Shot%202016-12-01%20at%204.23.41%20PM.png"
+Aseem kicked-off a new series of blog posts to showcase collectives with Open Source Design stay tuned for more to come.
```
Comment (asood123) [inline templates/emails/group.monthlyreport.text.hbs]: run on sentence. Need punctuation after "Open Source Design"

**Pull request merged**